scripts/findUnused.ts

```diff
@@ -99,52 +51,46 @@ for (const sourceFile of project.getSourceFiles()) {
 
             const sourceFileAndReferencesData = {
                 filePath: file,
-                sourceFileText,
+                text,
                 referencesLength: references.length,
                 declarationName: (
                     declaration as any as BindingNamedNode
                 ).getName()
             }
-            switch (declaration.getKindName()) {
-                case "VariableDeclaration":
-                    {
-                        const foundComment = checkForIgnoreUnusedComment({
-                            ...sourceFileAndReferencesData,
-                            declarationNode: declaration
-                                .getParentOrThrow()
-                                .getParentOrThrow()
-                        })
-
-                        if (foundComment) {
-                            return []
-                        }
-                    }
-                    break
-                default: {
-                    const foundComment = checkForIgnoreUnusedComment({
-                        ...sourceFileAndReferencesData,
-                        declarationNode: declaration
-                    })
+            if (declaration.isKind(SyntaxKind.VariableDeclaration)) {
+                const hasIgnoreComment = checkForIgnoreUnusedComment({
+                    ...sourceFileAndReferencesData,
+                    declarationNode: declaration.getFirstAncestorByKindOrThrow(
+                        SyntaxKind.VariableStatement
+                    )
+                })
+                if (hasIgnoreComment) {
+                    return []
+                }
+            } else {
+                const hasIgnoreComment = checkForIgnoreUnusedComment({
```

Comment:
This looks a bit repetitive with the `if` block. Let's instead take the approach where depending on whether the declaration is a variable declaration, we set a `declarationNode` variable to be either the original declaration or the `getFirstAncestorByKindOrThrow` result. Then we can just pass that to `checkForIgnoreUnusedComment` and handle both cases together since the rest of the logic is the same.